src/program.js

```diff
@@ -67,6 +69,13 @@ export class Program {
     name: string, description: string, executor: Function,
     commandOptions: Object = {}
   ): Program {
+    this.mapCommandToSubOpts = {};
+    this.mapCommandToSubOpts[name] = commandOptions;
```

Comment:
this doesn't look like it's used anywhere. I think you can delete `this.mapCommandToSubOpts`.